src/tests/e2e-test-recipes/DeviceInfoTests.json

```diff
@@ -1,102 +0,0 @@
-[
-  {
-    "Action": "LoadModule",
-    "Module": "deviceinfo.so"
-  },
-  {
```

Comment:
Please do not change these test recipes. We have lots more in temporarily disabled modules and not all align with this model. A test recipe can contain anything, there is no requirement to start with a certain header, etc. We do not want to put such restrictions.

Please consider only changing in the ASB recipe and keeping that as one single JSON file as it is.